### ISSUE TITLE
Dead end? 

### ISSUE THREAD

**Issue by kMikaZu** [simplerocket-llc/OpenCashew#1]
This seems to be a dead end?

**Comment by victoriafrench** [comment]
Seems to be so far... was just peeking at the code to see if I wanted to take it on... but this is a bit of a chore. It is written in swift 2.3 (and I don't even know if it's current state is working). It would need XCode 8 to upgrade to 3.0. Then XCode 9 to upgrade from 3.0 to 3.2. Then upgraded to 4 with each upgrade iteration fixing everything that the upgrader breaks.

But I am still looking at it. I would really like to use this app.

**Comment by slamorte** [comment]
I'd love this to work. It looked great for the two or three seconds it ran before freezing. This is a needed project.

**Comment by kMikaZu** [comment]
Yes and with Ship from RealArtist quitting as well... GitScout crashes as well. Bee is not as good and BugHub is way outdated. Not much options left... :-(

**Comment by slamorte** [comment]
I moved to zenhub.com. It's web-based, but I am pretty happy with it, and it works ok as a client interface to the issue list.

**Comment by kMikaZu** [comment]
Not very happy with the demo of zenhub. Rather slow, buggy MarkDown interface etc. Found Ship as a replacement for Cashew: they stopped developing. Found Bee as a replacement for Ship: they stopped developing. Seems to me there is no space for a decent mac GitHub issue client... :-( Very sad.

**Comment by bellebethcooper** [comment]
FYI, I spent *days* doing the upgrade to Swift 4 and then fixing crashes on launch that I found. It was a nightmare! I don't recommend it! Although maybe someone more experienced that me could do it faster.

Anyway, I'd like to maintain my own version of Cashew, as there are some things I'd like to change, so I've created a new repo for it here: https://github.com/bellebethcooper/cashew

If you'd like to keep something similar to what the Simple Rocket team already made, feel free to take the code from my repo and maintain it yourself separately. I've changed some stuff you might need to fiddle with, like the bundle identifier, which is why I haven't just done a PR here as well.

It's super early, so it runs, but it's very unstable. I'm discovering new crashes fairly often, and I'm really unconvinced that the log in code is stable, as I've had lots of weird issues with it. There's a lot to do to improve stability as well as fixing bugs and adding features I have in mind. I've also temporarily removed push notification support, because I'm going to try to achieve this with local notifications instead, and I've removed iCloud because I'm not sure why it's needed, and because I spent a good day trying to get it to work with no luck.

Eventually I'd like to make it available to simply download and install, if not pushing to the App Store, but for now you can compile the source and run it yourself hopefully. Feel free to jump into my repo with issues if you have questions, or if it doesn't run for you—I don't really know what I'm doing, but I want a good GitHub issues client, and I'm committed to keeping it open source so it doesn't die like all the rest!

**Comment by zenspider** [comment]
You might want to check out a newer app called Trailer. Has iOS and desktop variants.
